Cargo.toml

```diff
@@ -1,54 +1,12 @@
-[package]
-name = "auxmos"
-version = "2.0.0"
-authors = ["Putnam <putnam3145@gmail.com>"]
-edition = "2021"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
-[features]
-default = ["auxcleanup_gas_deletion", "turf_processing"]
-auxcleanup_gas_deletion = ["auxcleanup"]
-zas_hooks = []
-turf_processing = []
-fastmos = ["turf_processing"]
-katmos = ["fastmos"]
-katmos_slow_decompression = ["fastmos"]
-reaction_hooks = []
-plasma_fire_hook = ["reaction_hooks"]
-trit_fire_hook = ["reaction_hooks"]
-fusion_hook = ["reaction_hooks"]
-generic_fire_hook = ["reaction_hooks"]
-all_reaction_hooks = ["fusion_hook", "trit_fire_hook", "plasma_fire_hook", "generic_fire_hook"]
-
-[lib]
-crate-type = ["cdylib"]
-
-[dependencies]
-auxtools = { git = "https://github.com/willox/auxtools"}
-auxcallback = { git = "https://github.com/Putnam3145/auxcallback" }
-auxcleanup = { git = "https://github.com/Putnam3145/auxcleanup", optional = true}
-itertools = "0.10.3"
-rayon = "1.5.1"
-float-ord = "0.3.2"
-crossbeam = "0.8.1"
-flume = "0.10.11"
-parking_lot = "0.12.0"
-fxhash = "0.2.1"
-nonmax = "0.5.0"
-ahash = "0.7.6"
-lazy_static = "1.4.0"
-indexmap = { version = "1.8.0", features = ["rayon"] }
-dashmap = { version = "5.2.0", features = ["rayon"] }
-atomic_float = "0.1.0"
-petgraph = "0.6.2"
-bitflags = "1.3.2"
-
-[dependencies.tinyvec]
-version = "1.5.1"
-features = ["rustc_1_55", "alloc"]
+[workspace]
+members = [
+	"crates/auxmos",
+	"crates/auxcleanup",
+	"crates/auxcallback",
+]
 
 [profile.release]
-lto = 'thin'
+lto = 'fat'
```

Comment:
Why's this? Is there something funny going on here

Reply:
more perf, supposedly